.github/actions/test-dotnet/Invoke-SolutionTests.ps1

```diff
@@ -10,22 +10,25 @@ param ($Solution, $Verbosity, $Filter, $Configuration, $BlameHangTimeout)
 # the Actions web UI. Note that we use bash to output the log using bash to avoid pwsh wrapping the output to the
 # default buffer width.
 
-$connectionStringSuffix = @(
-    ';MultipleActiveResultSets=True;Connection Timeout=60;ConnectRetryCount=15;ConnectRetryInterval=5;Encrypt=false;'
-    'TrustServerCertificate=true'
-) -join ''
 if ($Env:RUNNER_OS -eq 'Windows')
 {
-    $connectionStringStem = 'Server=.;Database=LombiqUITestingToolbox_{{id}};Integrated Security=True'
+    $connectionSecurity = 'Integrated Security=True'
 }
 else
 {
-    $connectionStringStem = 'Server=.;Database=LombiqUITestingToolbox_{{id}};User Id=sa;Password=Password1!'
+    $connectionSecurity = 'User Id=sa;Password=Password1!'
 
     $Env:Lombiq_Tests_UI__DockerConfiguration__ContainerName = 'uitt-sqlserver'
 }
 
-$Env:Lombiq_Tests_UI__SqlServerDatabaseConfiguration__ConnectionStringTemplate = $connectionStringStem + $connectionStringSuffix
+$connectionString = @(
+    'Server=.;Database=LombiqUITestingToolbox_{{id}}'
+    $connectionSecurity
+    'MultipleActiveResultSets=True;Connection Timeout=60;ConnectRetryCount=15;ConnectRetryInterval=5;Encrypt=false'
+    'TrustServerCertificate=true'
```

Comment:
It seems quite arbitrary where you have semicolon and where it's a new string (the `TrustServerCertificate=true` on its own at the end). Why not break each component up?
```suggestion
    'Server=.'
    'Database=LombiqUITestingToolbox_{{id}}'
    $connectionSecurity
    'MultipleActiveResultSets=True'
    'Connection Timeout=60'
    'ConnectRetryCount=15'
    'ConnectRetryInterval=5'
    'Encrypt=false'
    'TrustServerCertificate=true'
```

Reply:
A bit of laziness 😶‍🌫️